src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  background-color: $color-white;
+  border: 1px solid $color-border;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;

```

Comment:
margin bottom.
If u want reset margin in p, add normalize.scss add reset all what u need there